### PR TITLE
feat: caco2_wang dataset

### PR DESCRIPTION
**Pull request by MicPie**
Add https://tdcommons.ai/single_pred_tasks/adme/#caco-2-cell-effective-permeability-wang-et-al

### CODE REVIEW

**Comment by MicPie** [comment]
[] check why yaml fails

File: data/caco2_wang/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+name: caco2_wang
+description: "The human colon epithelial cancer cell line, Caco-2, is used as an in\
+  \ vitro model\n        to simulate the human intestinal tissue. The experimental\
+  \ result on the rate of drug passing through\n        the Caco-2 cells can approximate\
+  \ the rate at which the drug permeates through the human intestinal tissue."
+targets:
+- id: permeability
+  description: Caco-2 cell effective permeability.
+  units: '?'
```
[the remark this answers is not in the document]
Comment (MicPie) [reply data/caco2_wang/meta.yaml]: Added correct unit

File: data/caco2_wang/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+          - compound id
+          - compound name
+license: CC BY 4.0
+links:
+    - url: https://tdcommons.ai/single_pred_tasks/adme/#caco-2-cell-effective-permeability-wang-et-al
+      description: tdcommons.ai dataset url
+    - url: https://pubs.acs.org/doi/10.1021/acs.jcim.5b00642
+      description: corresponding publication
+num_points: 910
+url: https://tdcommons.ai/single_pred_tasks/adme/#caco-2-cell-effective-permeability-wang-et-al
```
Comment (kjappelbaum) [inline data/caco2_wang/meta.yaml:34]:
```suggestion
url: https://tdcommons.ai/single_pred_tasks/adme/#caco-2-cell-effective-permeability-wang-et-al
```

Comment (kjappelbaum) [reply data/caco2_wang/meta.yaml:34]: It seems URL is duplicated. My fault as I didn't fix the check for this yet

**Review by kjappelbaum — CHANGES_REQUESTED** [review]
Thanks, two small comments. Otherwise, it looks good to me!

File: data/caco2_wang/meta.yaml (hunk 2 above)
Comment (MicPie) [reply data/caco2_wang/meta.yaml:34]: thanks for catching this, I removed one! :-)

File: data/caco2_wang/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      names:
+          - Caco-2 cell effective permeability
+          - Caco-2 cell permeability
+          - Caco-2 permeability
+          - permeability
+identifiers:
+    - id: SMILES
+      type: SMILES
+      description: SMILES
+    - id: compound_id
```
[the remark this answers is not in the document]
Comment (MicPie) [reply data/caco2_wang/meta.yaml:21]: Yes, I would keep it, some examples:
codeine, creatinine, danazol, (-)-epicatechin
but I renamed it to `compound_name`

**Pull request merged**